### PR TITLE
🆕🎨 Design updates #2

### PR DESCRIPTION
**Pull request by fairlighteth**
Waterfall PR to implement the new design.

- Updated logo loading functionality
- Added footer image (dark/light mode)
- Updated content/style for VERSIONS display (left bottom corner)
- Styling currency arrow separator logic
- Change 'Limit orders' -> 'Limit' as container title
- Styling input field background color
- Experimenting with lighter container colors on darkmode (cc @alfetopito ) - To be continued...

**Note:**
- Trying to gradually reduce the amount of theme variables we have. Forcing to use just a select few as possible.

**Current state:**

Screen Shot 2022-10-21 at 15 03 53

### CODE REVIEW

**Comment by alfetopito** [comment]
What are the changes here?
Can you please add a summary in the PR description? :)

File: src/custom/components/Version/index.tsx [hunk 1]
```diff
@@ -36,19 +32,25 @@ const VERSIONS: Record<
...
-        etherscan: _getContractsUrls(chainId, GP_VAULT_RELAYER),
-        github: `https://github.com/cowprotocol/contracts/blob/v${CONTRACTS_VERSION}/src/contracts/GPv2VaultRelayer.sol`,
-      }
+      // return Etherscan by default
+      return _getContractsUrls(chainId, GP_VAULT_RELAYER)
+
+      // return {
+      //   etherscan: _getContractsUrls(chainId, GP_VAULT_RELAYER),
+      //   github: `https://github.com/cowprotocol/contracts/blob/v${CONTRACTS_VERSION}/src/contracts/GPv2VaultRelayer.sol`,
+      // }
```
Comment (alfetopito) [inline src/custom/components/Version/index.tsx:41]: Is this only temporary or do you intent to remove the github link from the page?

Comment (fairlighteth) [reply src/custom/components/Version/index.tsx:41]: Yes my suggestion here is to pick one external link/url. IMO more links should be part of a FAQ or a dedicated page. But overall I think linking directly to the smart contract is best practice. Open to other suggestions here.

**Comment by fairlighteth** [comment]
> What are the changes here? Can you please add a summary in the PR description? :)

Added :)

**Comment by elena-zh** [comment]
Hey @fairlighteth , I know that I should not test it. just opened the PR based on my curiosity. Found some issues: 
1. Cows are everywhere now: 
cool!
cows everywhere
2. Mobile/tablet menu is in 'square containers'
menu in containers
3. Menu icons are too close to labels
icons to close in menu
4. Block and  gas price is not clickable
block not clickalbe
5. Vault relayer and Settlement contract should contains links to GH
image
6. Cows are not 100% hidden in a mobile/tablet views
cows in tablet


But anyways, the app looks great!!!

**Review by alfetopito — APPROVED** [review]
One point that might have been missed or still to be addressed.

Confirm modal on light mode is a bit weird

| dark | ligh |
| - | - |
| Screen Shot 2022-10-21 at 16 06 42 | Screen Shot 2022-10-21 at 16 06 17 |

File: src/cow-react/modules/application/containers/App/styled.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const HeaderWrapper = styled.div`
 
 export const FooterWrapper = styled(HeaderWrapper)`
   z-index: 1;
```
Comment (nenadV91) [inline src/cow-react/modules/application/containers/App/styled.ts:20]: I think you will have to set this z-index to `-1` if you want cows SVG behind the content.
But don't know if that has some hidden consequences. 
Screenshot from 2022-10-21 17-31-23

**Review by nenadV91 — APPROVED** [review]

**Pull request merged**